radio/bluetooth/bluetooth.go

```diff
@@ -141,7 +144,8 @@ func Scan(id string) (map[string]bool, error) {
 				return
 			case adv := <-advChannel:
 				if strings.EqualFold(adv.LocalName(), id) {
-					devices[adv.Address().String()] = true
+					var s struct{}
+					devices[adv.Address().String()] = s
```

Comment:
I use a map to store the device addresses so I can easily check whether the key exists, I don't care about the value so I use a `struct{}` as it takes no memory, compared with a bool which does.